sys/src/9/amd64/archamd64.c

```diff
@@ -30,8 +30,8 @@ cpuidinit(void)
 		return 0;
 	machp()->CPU.ncpuinfos++;
 
-	if(memcmp(&machp()->CPU.cpuinfo[0][1], "GenuntelineI", 12) == 0)
-		machp()->CPU.isintelcpu = 1;
+	//if(memcmp(&machp()->CPU.cpuinfo[0][1], "GenuineIntel", 12) == 0)
```

Comment:
Exactly that I was commenting in previous line-comment. It's a test, it's not used at any place in kernel. Probably and anciente debugging test or anything. It will be gone when WIP dissapeared in tittle.